src/modules/account.ts

```diff
@@ -3,7 +3,7 @@ import { ApiOptions, Contact } from '../models'
 import Snapshots from './snapshots'
 
 /**
- * Account is an API module for managing a wallet account
```

Comment:
Changing these all to node because we no longer have the wallet in this lib, and the term wallet might be confusing? Could be convinced to change these back tho.